shift_detector/precalculations/count_vectorizer.py

```diff
@@ -0,0 +1,87 @@
+import pandas as pd
+from shift_detector.precalculations.precalculation import Precalculation
+from nltk.corpus import stopwords as nltk_stopwords
+from sklearn.feature_extraction.text import CountVectorizer as CountVectorizer_sklearn
+from shift_detector.utils.column_management import ColumnType
+
+
+class CountVectorizer(Precalculation):
+
+    def __init__(self, cols, stop_words='english', max_features=None):
+        # potentially make min_df and max_df available
+        self.stopwords = None
+        self.max_features = None
+        self.cols = None
+        if isinstance(stop_words, str):
+            if stop_words in nltk_stopwords.fileids():
+                self.stop_words = nltk_stopwords.words(stop_words)
+            else:
+                raise Exception('The language you entered is not available')
+        elif isinstance(stop_words, list):
+            if all(isinstance(elem, str) for elem in stop_words):
+                for lang in stop_words:
+                    if lang not in nltk_stopwords.fileids():
+                        raise Exception('At least one language you entered is not available')
+                    if self.stopwords is None:
+                        self.stopwords = set(nltk_stopwords.words(lang))
+                    else:
+                        self.stopwords = self.stopwords.union(set(nltk_stopwords.words(lang)))
+            else:
+                raise TypeError('The stop_words list has to contain strings only')
+        else:
+            raise TypeError('Please enter the language for your stopwords as a string or list of strings')
+        if max_features is not None:
+            if isinstance(max_features, int):
+                if max_features > 0:
+                    self.max_features = max_features
+                else:
+                    raise ValueError('Max_features has to be at least 1')
+            else:
+                raise TypeError('Max_features has to be an int')
+        if cols is not None:
+            if isinstance(cols, list) and all(isinstance(col, str) for col in cols) or isinstance(cols, str):
+                self.cols = cols
+            else:
+                raise TypeError('Cols has to be list of strings or a single string')
```

Comment:
Columns may be integers, too. I think it's okay to not accept single strings, but only iterables

Reply:
I assume that columns shouldn't be integers, but I changed it so it should accept a list of strings only